.travis.yml

```diff
@@ -10,8 +10,10 @@ python:
 
 env:
     - DJANGO='1.8' REVERSION='1.9'  EXTRA=''
+    - DJANGO='1.8' REVERSION='1.10.1' EXTRA=''
     - DJANGO='1.9' REVERSION='1.10' EXTRA=''
     - DJANGO='1.8' REVERSION='1.9'  EXTRA='diff-match-patch'
+    - DJANGO='1.8' REVERSION='1.10.1'  EXTRA='diff-match-patch'
```

Comment:
Maybe, i don't know...

We test these combinations:
django 1.8.* with reversion 1.9.*
django 1.9.* with reversion 1.10.*

And all this with Python 2.7, 3.4 and 3.5
see here: https://travis-ci.org/jedie/django-reversion-compare/builds/106359851

Maybe it makes sense to add Django 1.8 with reversion 1.10.*
On the other side: This is more a job for reversion itself, isn't it? They run tests with older django versions too: https://travis-ci.org/etianen/django-reversion/builds/105372317

Think the current test matrix is fine, isn't it?